tests/cli/test_cli.py

```diff
@@ -517,3 +518,20 @@ def test_run_cron_elapsed(runner, tmp_path):
 
     assert result.exit_code == 0
     assert_model_batches_executed(result)
+
+def test_clean(runner, tmp_path):
+    # Create and backfill `prod` environment
+    create_example_project(tmp_path)
+    init_prod_and_backfill(runner, tmp_path)
+
+    # Confirm cache exists
+    cache_path = Path(tmp_path) / ".cache"
+    assert cache_path.exists()
+    assert len(list(cache_path.iterdir())) > 0
+
+    # Invoke the clean command
+    result = runner.invoke(cli, ["--paths", tmp_path, "clean"])
+
+    # Confirm cache was cleared
+    assert result.exit_code == 0
```

Comment:
This is the operational bit here. We could remove the Path based assertions before and after and eliminate the extra import if preferred. I added it here to verify the test setup.

Reply:
I think this is great